### PR TITLE
Create Marketplace Guide page for Native Mobile AR 

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

**Review by HedwigAR — APPROVED** [review]

**Comment by Adam-Dupaski** [comment]
@Luyao-Zhang-1 @HedwigAR is it possible to get some more substantial documentation here? Various sections from the template could be considered: https://github.com/mendix/docs/blob/development/templates/mg-page-template.md
CC: @ConnorLand

**Comment by Luyao-Zhang-1** [comment]
> @Luyao-Zhang-1 @HedwigAR is it possible to get some more substantial documentation here? Various sections from the template could be considered: https://github.com/mendix/docs/blob/development/templates/mg-page-template.md
> CC: @ConnorLand

@Adam-Dupaski Hegwig will have more time to look into this in the next sprint. We will work on this together after I am back from my vacation.

**Comment by Adam-Dupaski** [comment]
@Luyao-Zhang-1 okay then we can publish this now and it will receive further updates. Please create a Jira issue for these updates.

**Comment by Luyao-Zhang-1** [comment]
> @Luyao-Zhang-1 okay then we can publish this now and it will receive further updates. Please create a Jira issue for these updates.

@Adam-Dupaski Before we publish it, could you help check the YAML header of the file?

File: content/appstore/modules/native-mobile-ar.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Native Mobile AR"
+category: "Modules"
+description: "Describes the Native Mobile AR module, which is available in the Mendix Marketplace."
+tags: ["marketplace", "marketplace component", "native mobile resources", "platform support"]
+---
+
+## 1 Introduction
+
+The [Native Mobile AR](https://marketplace.mendix.com/link/component/117209) module contains the widgets for building AR experience with Native Mobile. This module is also part of the [AR Starter App](https://marketplace.mendix.com/link/component/117211) that contains examples.
```
Comment (Adam-Dupaski) [inline content/appstore/modules/native-mobile-ar.md:10]: @Luyao-Zhang-1 @ConnorLand when speaking of "native mobile" generically, we should not capitalize, correct? If that's the case, should we add this to the Docs SG?

Comment (Luyao-Zhang-1) [reply content/appstore/modules/native-mobile-ar.md:10]: @Adam-Dupaski @ConnorLand I checked in our docs, indeed "native mobile" are lower-case when we use them generically. When they are part of a specific term, for example "Mendix Native Mobile Builder", we capitalize the first letters. Should I add this to the agenda of our docs meeting today?

Comment (Adam-Dupaski) [reply content/appstore/modules/native-mobile-ar.md:10]: @Luyao-Zhang-1 change above then please, and add it to the terminology section of the SG. No need to discuss in the meeting.

Comment (Luyao-Zhang-1) [reply content/appstore/modules/native-mobile-ar.md]: @Adam-Dupaski Done.

**Review by Adam-Dupaski — APPROVED** [review]

File: content/appstore/modules/native-mobile-ar.md (hunk 1 above)
Comment (ConnorLand) [reply content/appstore/modules/native-mobile-ar.md]: This is correct.

**Comment by ConnorLand** [comment]
> @Luyao-Zhang-1 @HedwigAR is it possible to get some more substantial documentation here? Various sections from the template could be considered: https://github.com/mendix/docs/blob/development/templates/mg-page-template.md
> CC: @ConnorLand

This sounds like a great idea. Shall I put a helpful set of eyes on whatever Luyao and Hedwig come up with?

**Pull request merged**